docs/plugins/profile.md

```diff
@@ -1,150 +1,151 @@
+Occasionally, your App requires profile information about the current user. This information is usually provided by the operator or platform.
 
-Since each operator / platform may have a different implementation, the Lightning SDK offers a Profile plugin with a generic interface to the developer.
+Because each operator or platform might implement user profile information in a different way, the Lightning SDK provides the *Profile* plugin. This plugin is a generic interface for developers, independent of any operator or platform.
 
-The Profile plugin also offers the ability to update profile information.
+You can also use the Profile plugin to *update* profile information.
 
 ## Usage
 
-Whenever you need Profile information, import the Profile plugin from the Lightning SDK
+If you need Profile information, import the Profile plugin from the Lightning SDK:
 
 ```js
 import { Profile } from '@lightningjs/sdk'
 ```
 
-## Available methods
+## Available Methods
 
-When you call a method without params it will _return_ the Profile information in the form of a promise.
-If you pass params, it will _update_ the Profile information.
+If you call one of the methods below *without* parameters, it returns the Profile information in the form of a *promise*. If you pass parameters, the method *updates* the Profile information.
 
 ### ageRating
 
-Gets the ageRating. Returns `adult` by default during _local development_.
+Gets the user's age rating. Returns `adult` by default during *local development*.
 
 ```js
 Profile.ageRating()
 ```
 
-### City
+### city
 
-Gets the city. `New York` by default during _local development_.
+Gets the user's city. Returns `New York` by default during *local development*.
 
 ```js
 Profile.city()
 ```
 
+### zipCode
 
-### ZipCode
-
-Gets the zipCode. Returns `27505` by default during _local development_.
+Gets the user's zip code. Returns `27505` by default during *local development*.
 
 ```js
 Profile.zipCode()
 ```
 
-### CountryCode
+### countryCode
 
-Gets the countryCode. Returns `US` by default during _local development_.
+Gets the user's country code. Returns `US` by default during *local development*.
 
 ```js
 Profile.countryCode()
 ```
 
-### Ip
+### ip
 
-Gets the ip. Returns `127.0.0.1` by default during _local development_.
+Gets the user's IP address. Returns `127.0.0.1` by default during *local development*.
 
 ```js
 Profile.ip()
 ```
 
-### Household
+### household
 
-Gets the houseHold-id. Returns `b2244e9d4c04826ccd5a7b2c2a50e7d4` by default during _local development_.
+Gets the user's household ID. Returns `b2244e9d4c04826ccd5a7b2c2a50e7d4` by default during *local development*.
 
 ```js
 Profile.household()
 ```
 
-### Language
+### language
 
-Gets the language. During _local development_ will attempt to return the browser's language, with a default to `en`.
+Gets the user's mother language. During *local development*, it attempts to return the browser's language with `en` as default.
```

Comment:
not sure if mother language is appropriate here?

Reply:
Suggestion for "Gets the user's mother language." => "Gets the user's language, based on the currently set locale."

Reply:
yes, that sounds like a good change